scripts/check_backend_associated_test_file_test.py

```diff
@@ -44,19 +44,19 @@ def mock_error(msg: str) -> None:
 
     def test_checks_fail_when_a_backend_file_lacks_associated_test_file(
             self) -> None:
-        tempdir = tempfile.TemporaryDirectory(prefix=os.getcwd() + '/core/')
-        backend_file = os.path.join(tempdir.name, 'backend_file.py')
-        frontend_file = os.path.join(tempdir.name, 'frontend_file.ts')
+        with tempfile.TemporaryDirectory(
+            prefix=os.getcwd() + '/core/') as tempdir:
```

Comment:
```suggestion
        with tempfile.TemporaryDirectory(
            prefix=os.getcwd() + '/core/'
        ) as tempdir:
```

Ensuring that the argument to `with` ends at the same indentation level as `with` makes it easier to identify the block of code affected by the context manager. Ditto elsewhere

Reply:
You're absolutely right, this is more readable. Have adjusted the indentation throughout my changes.